java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CompositeConfigService.java

```diff
@@ -20,31 +22,14 @@
  */
 @Builder
 @RequiredArgsConstructor(access = AccessLevel.PACKAGE)
-public class CompositeConfigService implements ConfigService {
+public class CompositeConfigService implements ConfigService, SecretStore {
```

Comment:
```suggestion
public class CompositeConfigService implements SecretStore {
```
Should be enough too